--- a/src/plugins/tileseditor/tileseditoreffect.cpp
+++ b/src/plugins/tileseditor/tileseditoreffect.cpp
@@ -103,17 +103,6 @@
     return 70;
 }
 
-void TilesEditorEffect::grabbedKeyboardEvent(QKeyEvent *keyEvent)
-{
-    if (m_toggleShortcut.contains(keyEvent->key() | keyEvent->modifiers())) {
-        if (keyEvent->type() == QEvent::KeyPress) {
-            toggle();
-        }
-        return;
-    }
-    QuickSceneEffect::grabbedKeyboardEvent(keyEvent);
-}
-
 } // namespace KWin
 
 #include "moc_tileseditoreffect.cpp"
--- a/src/plugins/tileseditor/tileseditoreffect.h
+++ b/src/plugins/tileseditor/tileseditoreffect.h
@@ -28,7 +28,6 @@
     void setAnimationDuration(int duration);
 
     int requestedEffectChainPosition() const override;
-    void grabbedKeyboardEvent(QKeyEvent *keyEvent) override;
 
 public Q_SLOTS:
     void toggle();
